kubernetes/check-pods.sh

```diff
@@ -0,0 +1,32 @@
+#! /bin/bash -e
+
+pod_label=$1
+statefulset=$2
+
+# Statefulset starts the pods individually, but with the convention `{statefulset-name}-{0-n}`
+# The second member {0-n} corresponds to the amount of pods
+# We have to manually defined name of the pods based of amount of replicas.
+if [[ "$statefulset" == "true" ]]; then
+    pods_name=()
+    for i in `seq 0 $(($(kubectl get statefulset $pod_label -o jsonpath='{.spec.replicas}')-1))`; do
+        pods_name+=(pods/$pod_label-$i)
+    done
+    [ -z "$pods_name" ] && echo "The pod does not exist!" && exit 1
+else
+    pods_name=$(kubectl get pod -l "run=$1" -o name)
```

Comment:
You're relying on the `run` label here. You should explain at the beginning of the script that it only works with pods that have this label.

Reply:
Done.